packages/sdk/webpack.config.ts

```diff
@@ -8,7 +8,7 @@ const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
 
 function configFor(target: 'web' | 'node' | 'react-native'): webpack.Configuration {
     const t = target === 'react-native' ? 'web' : target;
-    const entry = resolve(__dirname, './src/index.ts');
+    const entry = [resolve(__dirname, './src/index.ts'), resolve(__dirname, './src/pub/plt.ts')];
```

Comment:
Nice. That means we can simplify this testing setup again, isn't it?

https://github.com/Concordium/concordium-desktop-wallet/blob/main/jest.config.ts#L20

Reply:
I guess it would, yes 😄